Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM debian:jessie
```

Comment:
`pandora.inf.elte.hu` (the recommended internal development server) is on Debian stretch—and it works—, so maybe upgrade to it? The installed versions of Flex and Bison are older, however (2.6.1 and 3.0.4, respectively), and CMake and Ninja aren’t installed at all:
```console
$ dpkg-query -l  | grep -P '(bison|flex|cmake|ninja)'
ii  bison                             2:3.0.4.dfsg-1+b1                 i386         YACC-compatible parser generator
ii  bisonc++                          5.02.00-1                         i386         Bison-style parser generator for C++
ii  flex                              2.6.1-1.3                         i386         fast lexical analyzer generator
ii  libbison-dev:i386                 2:3.0.4.dfsg-1+b1                 i386         YACC-compatible parser generator - development library
ii  libfl-dev:i386                    2.6.1-1.3                         i386         static library for flex (a fast lexical analyzer generator)
$ which cmake
$ which ninja
```